src/main/java/at/hannibal2/skyhanni/features/garden/SensReducer.kt

```diff
@@ -0,0 +1,119 @@
+package at.hannibal2.skyhanni.features.garden
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.ConfigLoadEvent
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.KeyboardManager.isKeyHeld
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.afterChange
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import net.minecraft.client.Minecraft
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+object SensReducer {
+    private val config get() = SkyHanniMod.feature.garden.sensReducerConfig
+    private val storage get() = SkyHanniMod.feature.storage
+    private var isToggled = false
+    private var isManualToggle = false
+    private var lastCheckCooldown = SimpleTimeMark.farPast()
+
+    private val mc get() = Minecraft.getMinecraft()
+    private val gameSettings = mc.gameSettings
+
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
```

Comment:
`LorenzTickEvent` only gets called when inside the world. This means quitting out while having reduced sensitivity will keep that reduced sensitivity accross restarts, and, since the current set up "forgets" that it was toggled on, does not restore the sensitivity after a restart.